src/pot.sol

```diff
@@ -129,6 +129,7 @@ contract Pot is DSNote {
     }
 
     function cage() external note auth {
+        require(live == 1);
```

Comment:
why do we need this?

Reply:
The only one that seems necessary to prevent a double running of `cage` is in the `vow` (which is added), but here is not necessary.

Reply:
Fair enough, this one really isn't necessary, I guess I was just feeling extra paranoid when I worked on this file :). Will remove it.

Reply:
done